src/main/java/bio/terra/cloudfunctions/common/CloudEventsHarness.java

```diff
@@ -68,13 +68,13 @@ private void parseCloudEventV1() throws Exception {
         message =
             new FirestoreEventMessage(
                 event.getData().toBytes(),
-                d -> GsonWrapper.getInstance().fromJson(new String(d), DocumentEventData.class));
+                d -> GsonWrapper.convertFromClass(new String(d), DocumentEventData.class));
```

Comment:
I still don't see the purpose of supporting all these event types. What is the benefit here?

Reply:
This class will be shelved when we do a final merge with Drew's PR#8. You're right, we don't need to support these event types.